packages/svelte/src/compiler/types/template.d.ts

```diff
@@ -457,6 +457,11 @@ export type Block = EachBlock | IfBlock | AwaitBlock | KeyBlock | SnippetBlock;
 export interface Attribute extends BaseNode {
 	type: 'Attribute';
 	name: string;
+	/**
+	 * A boolean attribute is either present (true) or not.
+	 * Quoted/string values are represented as an array of alternating text and expressions whether
+	 * complex (`"foo{x}bar"`) or holding just a single expression like `"{x}"`.
```

Comment:
This isn't accurate — they don't need to be alternating (`"{a}{b}"` is just two consecutive `ExpressionTag` nodes). We already know what they're an array _of_, it's right there in the type definition

```suggestion
	 * Quoted/string values are represented by an array, even if they contain a single expression like `"{x}"`
```